README.md

```diff
@@ -1,10 +1,10 @@
 <!--badges-->
 [![License: CC BY-IGO 3.0](https://licensebuttons.net/l/by-nc/3.0/igo/80x15.png)](https://creativecommons.org/licenses/by/3.0/igo)
-![CI Build](https://img.shields.io/github/actions/workflow/status/WorldHealthOrganization/smart-trust/ghbuild.yml)  
+![CI Build](https://img.shields.io/github/actions/workflow/status/costateixeira/smart-trust/ghbuild.yml)  
```

Comment:
@costateixeira is this change intended to refer to your repo? Pls review the Readme file changes